tests/test_doc_examples.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+import yaml
+from parameterized import parameterized
+import itertools
+
+from c7n.resources import load_resources
+from c7n.exceptions import PolicyValidationError
+from c7n.provider import resources
+from .common import BaseTest
+
+
+def get_doc_examples():
+    policies = []
+    for key, v in resources().items():
+        #filters
+        for k, filter in v.filter_registry._factories.items():
```

Comment:
instead of poking at the internal dict, just use .items() on the filter and action registries, like you did with the resource registry.

Reply:
good catch